src/models/tasks/task.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+from src.common.database import Database
+from src.models.folders import constants
+
+
+class Task(object):
+    def __init__(self, title, description, due_date, folder, user_id, _id=None):
+        self.title = title
+        self.description = description
+        self.due_date = due_date
+        self.folder = folder
```

Comment:
I am just curious (apologies if I am bugging) the folder is basically an object passed correct? or is the folder_id?
If is the object do you think is best to use the whole object? I am thinking we only need the folder_id  since other attributes of folder are not important in task like description and such? what do you think? In this sense task is created from inside of folder. Since to task can exist without a folder. What do you say?